swap/swap_test.go

```diff
@@ -866,21 +866,21 @@ func TestSaveAndLoadLastReceivedCheque(t *testing.T) {
 }
 
 // newTestSwapAndPeer is a helper function to create a swap and a peer instance that fit together
-// the owner of this swap is the beneficiaryAddress
-// hence the owner of this swap would sign cheques with beneficiaryKey and receive cheques from ownerKey (or another party) which is NOT the owner of this swap
+// the issuer of this swap is the beneficiaryAddress
+// hence the issuer of this swap would sign cheques with beneficiaryKey and receive cheques from issuerKey (or another party) which is NOT the issuer of this swap
```

Comment:
same as above: not convinced `swap` or `contract` can use the terminology of `issued` like cheques can

Reply:
The confusion here has in my opinion not so much to do with renaming `owner` to `issuer`. 

It underlines, IMO, that we have created somewhat of confusin situation due to naming:

Here we see again that an issuer can be also a beneficiary and vice versa. What makes it extra confusing here, is that in the Swarm source code (`protocol.go:124` and `peer.go:34`), we speak about the `peer.beneficiary` as the counterparty and we set our own address and contract in the `swap` struct. Whereas here, we are just doing it the other way around, saving the `issuer` in the peer struct and the beneficiary in the `swap` struct.

We have 4 parties to name properly:

- Counterparty Externally Owned account,
Currently `peer.beneficiary`
- Counterparty chequebook address,
Currently `peer.contractAddress`
- Our own Externally Owned account,
Currently `swap.issuer.address`
- Our own chequebook address
Currently `swap.issuer.Contract`

What do you think: is it worth spending time on attempting to find perfect names for those, or shall we just leave as it is, and accept that there can be ambiguity and that we risk making mistakes as in the function `newTestSwapAndPeer`?

Reply:
I understand the cause of confusion, should I leave alone for now?.

Or should I refactor the comments?, I'm a bit confused.

Reply:
I would leave this one for now. I will bring it up as a point on the agenda for our next sync meeting!